.github/workflows/container_build_fix.yml

```diff
@@ -0,0 +1,36 @@
+# this workflow can be triggered manually to initiate any build if it fails
+name: fix container build
+
+"on":
+  workflow_dispatch:
+    inputs:
+      container_name:
+        type: choice
+        required: true
+        description: "select image name to build"
+        options: [universal]
```

Comment:
any reason why we limit this?

Reply:
I want to start small and limit this to universal only. We'll add base and devel later, once I test this on my own repo.